### PR TITLE
Ir table

### PR DESCRIPTION
**Pull request by Broham**
Moving the Incident Report table to it's own template.  Reference the new template from the suggest incidents modal _and_ the curator inbox now

### CODE REVIEW

File: client/controllers/curatorInbox/curatorSourceDetails.coffee [hunk 1]
```diff
@@ -29,6 +29,7 @@ _getSource = (instance, sourceId) ->
   instance.source.set source
 
 Template.curatorSourceDetails.onCreated ->
+  @incidentCollection = new Meteor.Collection(null)
```
Comment (jgoley) [inline client/controllers/curatorInbox/curatorSourceDetails.coffee]: This may not be necessary since its declared later in the autorun?

Comment (Broham) [inline client/controllers/curatorInbox/curatorSourceDetails.coffee]: Yep - good call

**Comment by jgoley** [comment]
Looks like on the extract-incidents route the table appears when the source is processing.
Also looks like you will need to pass in at least the articles? There is an error when clicking the case count which should open the modal to edit the IR. You may need to transfer some events and controller logic to the new template?

**Comment by jgoley** [comment]
`Uncaught TypeError: this.data.articles.fetch is not a function`
Or maybe the articles is an array instead of a collection?

**Comment by Broham** [comment]
@jgoley - good catch.  I ended up passing the flag that handles the showing/hiding of the IR table to the template itself.  Look better?

**Comment by Broham** [comment]
Ok error should be resolved @jgoley - want to give it one more review before I merge?

**Comment by Broham** [comment]
Ahh ok @jgoley - where else are you seeing it?

**Comment by jgoley** [comment]
Lines 51 and 85 in the incidentForm controller.

**Comment by Broham** [comment]
ok @jgoley - i pulled those out as well

**Comment by jgoley** [comment]
👍  Looks good

**Pull request merged**